build.sbt

```diff
@@ -118,7 +118,7 @@ lazy val elasticsearch = project.in(file("extensions/elasticsearch"))
     name := "akka-stream-extensions-elasticsearch",
     libraryDependencies ++= Seq(
       "com.typesafe.akka" %% "akka-stream" % "2.4.2",
-      "org.elasticsearch" % "elasticsearch" % "1.3.2"
+      "org.elasticsearch" % "elasticsearch" % "2.4.0"
```

Comment:
Hi! I think it would be better to not include elastic to build. This is achieved by adding `% "provided,test"` to dependency expression